File: 231 Power of Two/solution.py
class Solution:
    # @param {integer} n
    # @return {boolean}
    def isPowerOfTwo(self, n):
        if n==0:
            return False
        result = 0
        while (n!=1):
            n,result = divmod(n,2)
            if result ==1:
                return False
        return True
        

File: 231 Power of Two/solution.cpp
class Solution {
public:
    bool isPowerOfTwo(int n) {
        if (n==0)
            return false;
        int result = 0;
        while (n!=1){
            result = n%2;
            n = n/2;
            if (result)
                return false;
        }
        return true;
    }
};